var/spack/repos/builtin/packages/trimal/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Trimal(MakefilePackage):
+    """A tool for automated alignment trimming in large-scale
+       phylogenetic analyses"""
+
+    homepage = "https://github.com/scapella/trimal"
+    url      = "https://github.com/scapella/trimal/archive/v1.4.1.tar.gz"
+
+    version('1.4.1', sha256='cb8110ca24433f85c33797b930fa10fe833fa677825103d6e7f81dd7551b9b4e')
+
+    build_directory = 'source'
+
+    def install(self, sinstall_treepec, prefix):
```

Comment:
`sinstall_treepec` is likely a typo of `spec`. That doesn't affect the installation though since the variable (or `spec`) is not referenced

Reply:
Oops. Did not notice that! Think it was a sneaky vim moment.